### PR TITLE
Create an order for a selected service

### PR DESCRIPTION
**Pull request by julianoviedo1**
Orders Form View:

image

Still needs back-end features

### CODE REVIEW

File: app/controllers/services_controller.rb [hunk 1]
```diff
@@ -1,7 +1,10 @@
 class ServicesController < ApplicationController
   before_action :set_service, only: %i[show]
 
-  def show; end
+  def show
+    @order = Order.new
+    # @order = @service.orders.new
```
Comment (julianoviedo1) [inline app/controllers/services_controller.rb]: This has not worked, but i think it should. Isnt it?

Comment (julianoviedo1) [reply app/controllers/services_controller.rb]: Solved

**Comment by julianoviedo1** [comment]
Form result:

image

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -1,4 +1,53 @@
...
+  def show; end
+
+  def edit; end
+
+  def create
+    @order = Order.new(order_params)
+    @order.user = current_user
+    @order.service_id = params[:service_id]
+    @order.payment = current_user.payments.first
+    @order.status = 'pending'
```
Comment (jpbalarini) [inline app/controllers/orders_controller.rb]: there's too much logic in this method. The best thing would be to create a private method to do all this. Something like `build_order` maybe?

File: app/views/services/show.html.erb [hunk 3]
```diff
@@ -15,17 +15,16 @@
     </div>
   </section>
   <section class='container w-75 bg-white'>
+    <%= simple_form_for @order, url: service_orders_path(@service) do |f| %>
     <h6 class="fw-bold letter-spacing py-4 text-dark fs-6 mx-4">INVITADOS</h6>
```
Comment (jpbalarini) [inline app/views/services/show.html.erb]: indentation doesn't look good here (should be inside the form)

**Review by jpbalarini — CHANGES_REQUESTED** [review]

File: app/views/services/show.html.erb (hunk 3 above)
Comment (julianoviedo1) [reply app/views/services/show.html.erb]: Resoslved in commit: 83ad4ae

File: app/controllers/orders_controller.rb (hunk 2 above)
Comment (julianoviedo1) [reply app/controllers/orders_controller.rb]: Resolved in commit: 96a6668

**Review by jpbalarini — APPROVED** [review]
👍

**Pull request merged**